### ISSUE TITLE
PCO:carrying capacity to cede to ECOCORE

### ISSUE THREAD

**Issue by kaiiam** [PopulationAndCommunityOntology/pco#62]
Hello @ramonawalls . I would like to at some point, cede the PCO:[carrying capacity](http://purl.obolibrary.org/obo/PCO_0000009) to [ECOCORE](https://github.com/EcologicalSemantics/ecocore) the ontology of core ecological entities recently created with @pbuttigieg as collaborator. 

I believe it should be housed in ECOCORE as carrying capacity is a quality of an ecosystem rather than of a population, for which propose the new definition:
```
A quality inherent in an an ecosystem, in which a population residing within and depending upon such ecosystem, is restricted to a maximum sustainable population size.
```
However since ECOCORE is not yet hosted on Ontobee, I'm hesitant to change the purl if that means removing it from current usage.  Could I perhaps update the term in PCO, make it a subclass to a new term: **quality of an ecosystem** and add an editors note to be ceded to ECOCORE pending ECOCORE's addition to Ontobee?

Cheers,
Kai

**Comment by ramonawalls** [comment]
@kaiiam I agree that carrying capacity is not a quality of a population, but rather of an ecosystem, and that ecocore is a better home. However, I am reluctant to add quality of an ecosystem to PCO, as it will just be deprecated and moved to ecocore. Also, a substantive change to carrying capacity (i.e. changing its parent term) requires that it be deprecated and replaced by a new term.  I think a better solution would be to deprecate the PCO term, with a note that is moving to ecocore, then go ahead and create the new terms in ecocore. As far as I know, no large projects are using carrying capacity from PCO. If they were, they probably would have pointed out that it was wrong!

**Comment by ramonawalls** [comment]
@kaiiam Just wondering if anything ever happened with this? Should the issue remain open?
